docs/data/material/components/buttons/InputFileUpload.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+import CloudUploadIcon from '@mui/icons-material/CloudUpload';
+
+const VisuallyHiddenInput = styled('input')`
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  white-space: nowrap;
+  width: 1px;
+`;
```

Comment:
As far as I know, the standard is for all the demos should use the JavaScript object syntax in Base UI, Joy UI, Material UI.